text/pvm.tex

```diff
@@ -575,7 +575,7 @@ \subsubsection{Instruction with Arguments of Two Registers and Two Immediates}
     \reg'_B \equiv \reg'_{r_B} \\
     \using l_X &= \min(4, \instructions_{\imath+2} \bmod 8) \,,\quad&
     \immed_X &= \sext_{l_X}(\de_{l_X}(\instructions_{\imath+3\dots+l_X})) \\
-    \using l_Y &= \min(4, \max(0, \ell - l_X - 2)) \,,\quad&
+    \using l_Y &= \min(4, \max(0, \ell - l_X - 3)) \,,\quad&
```

Comment:
Incorrect. $\ell$ gives the skip distance: the number of bytes in the PVM code to skip *not including the instruction byte*. We therefore substract 2 (one for registers, the other for first immediate length) plus the length of the first immediate, to end up with the remaining skip bytes for the length of the second immediate.

Reply:
Apologies, this slipped in due to a previous conversation with Jan. It should not have been part of this PR.